src/common/query/src/logical_plan/expr.rs

```diff
@@ -18,6 +18,33 @@ use common_time::Timestamp;
 use datafusion_common::{Column, ScalarValue};
 use datafusion_expr::expr::Expr;
 use datafusion_expr::{and, binary_expr, Operator};
+use datatypes::data_type::DataType;
+use datatypes::schema::ColumnSchema;
+use datatypes::value::Value;
+
+pub fn build_same_type_ts_filter(
```

Comment:
Could you add a comment about this function? Thank you. It's better to have a comment for every public function.